Capfile

```diff
@@ -18,9 +18,13 @@ require 'capistrano/deploy'
 # require 'capistrano/rvm'
 # require 'capistrano/rbenv'
 # require 'capistrano/chruby'
-require 'capistrano/bundler'
+# require 'capistrano/bundler'
+
 require 'capistrano/rails'
 require 'dlss/capistrano'
+require 'capistrano/passenger'
+# FIXME: this isn't working for dev or stage:
+# 'SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed'
```

Comment:
Should we remove this comment now?  I believe this issue has been resolved by getting a proper cert on the squash server (and we'll be moving to honeybadger anyway).

Want to give this a retry now that the cert issue should be resolved?